frontend/app/model/Schema.scala

```diff
@@ -25,8 +25,17 @@ case class OfferSchema(
  category: String,
  price: String,
  priceCurrency: String,
+ validFrom:String,
  availability: Option[String],
  `@type`: String = "Offer")
+object OrganizerSchema {
+  implicit val writesSchema = Json.writes[OrganizerSchema]
+}
+
+case class OrganizerSchema(
```

Comment:
The spacing/formatting seems a little off compared to the other case classes in this file, eg. the props of the class are overly indented. It'd be better for readability if it was the same  as the rest of the module.

Reply:
Reformatted for readability